cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowToken.java

```diff
@@ -42,7 +42,10 @@
   /**
    * Put the specified key and value into the {@link WorkflowToken}.
    * The token may store additional information about the context in which
-   * this key is being set, for example, the unique name of the workflow node.
+   * this key is being set, for example, the unique name of the workflow node
```

Comment:
Oh ok.  I still think its confusing because the sentence is long. What about just breaking it into two:

this key is being set. For example, the token may store the name of the workflow node that is performing the operation, or the name of the workflow if the operation is performed in initialize or destroy method.